src/app/core/functions/flatten.ts

```diff
@@ -0,0 +1,26 @@
+export const flatten = (target: object) => {
```

Comment:
can you jsdoc this function ?
example: https://github.com/iq180-netcentric/core-logic/blob/master/iq180.ts

Reply:
After the tomorrow's quiz.